src/Utilities/Utils.js

```diff
@@ -7,4 +7,23 @@ export const cleanAllMovies = (movies) => {
             "backdrop": movie.backdrop_path
         }
     })
+}
+
+export const modifyMovieData = (movie) => {
```

Comment:
Wow! This is really thorough cleaning. Thank you for reformatting the data like this!